debezium-connector-mysql/src/test/docker/init-alt-replica/setup.sql

```diff
@@ -6,12 +6,14 @@
 -- 2) 'snapper' - all privileges required by the snapshot reader AND binlog reader
 -- 3) 'mysqluser' - all privileges
 --
-GRANT REPLICATION SLAVE, REPLICATION CLIENT ON *.* TO 'replicator' IDENTIFIED BY 'replpass';
-GRANT SELECT, RELOAD, SHOW DATABASES, REPLICATION SLAVE, REPLICATION CLIENT  ON *.* TO 'snapper'@'%' IDENTIFIED BY 'snapperpass';
+CREATE USER 'replicator' IDENTIFIED BY 'replpass';
```

Comment:
@jpechane Why are these changes needed? Should we add something to the docs, too, regarding permissions needed by the connector?

Reply:
@gunnarmorling There is no difference in permitions - just the user can no longer be implictly created by `GRANT` but must be created exaplicitly.

Reply:
Ah, yes. Thx!